shared/util/forward-logs.js.flow

```diff
@@ -2,6 +2,39 @@
 export const setupSource = () => {}
 export const setupTarget = () => {}
 
-export const localLog = (...args: any) => {}
-export const localWarn = (...args: any) => {}
-export const localError = (...args: any) => {}
+export const localLog = (
```

Comment:
This feels a bit gross to me, and I don't think it's necessary if we merge https://github.com/keybase/client/pull/7593